mock-caas/src/main/java/com/sequenceiq/caas/grpc/service/MockUserManagementService.java

```diff
@@ -162,6 +172,60 @@ public GetEventGenerationIdsResponse load(String key) throws Exception {
                         return respBuilder.build();
                     }
                 });
+
+        initializeActorWorkloadCredentials();
+    }
+
+    private void initializeActorWorkloadCredentials() {
+        final int keyType17 = 17;
+        final int keyType18 = 18;
+        final int saltType = 4;
+        final int hashsaltlength = 32;
+        final int keysaltlength = 8;
+        final String password = "Password123!";
+        final String saltprefix = "{SSHA512}";
+
+        try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
+
+            byte[] salt = RandomStringUtils.randomAscii(hashsaltlength).getBytes(Charsets.UTF_8);
+            MessageDigest md = MessageDigest.getInstance("SHA-512");
+            md.update(password.getBytes(Charsets.UTF_8));
+            md.update(salt);
+            baos.write(md.digest());
+            baos.write(salt);
+            String hash = saltprefix + Base64.encodeBase64String(baos.toByteArray());
+
+            String userName = RandomStringUtils.randomAlphanumeric(keysaltlength);
+            String realm = RandomStringUtils.randomAlphanumeric(keysaltlength);
+            String principalName = userName + '@' + realm;
+            String keySalt = realm + userName;
+            KerberosPrincipal principal = new KerberosPrincipal(principalName);
+            KerberosKey keyAES128 = new KerberosKey(principal, password.toCharArray(), "AES128");
+            KerberosKey keyAES256 = new KerberosKey(principal, password.toCharArray(), "AES256");
```

Comment:
If we only need to care about there being a single password here (Password123) why don't we just precompute the value and stick it in as a constant? That seems easier than duplicating all the code from UMS.

Reply:
yeah, let me explore and get the actual coming from UMS and use that.

Reply:
@toddlipcon Thanks. I have changed for JSON and parsing it directly into the response. 
Unit test is only for checking if the values are getting loaded or not.